tests/framework/extensions/rke1/nodepools/nodepools.go

```diff
@@ -107,38 +65,54 @@ func ScaleWorkerNodePool(client *rancher.Client, nodeRoles []NodeRoles, ClusterI
 		ControlPlane:            false,
 		DeleteNotReadyAfterSecs: 0,
 		Etcd:                    false,
-		HostnamePrefix:          "auto-rke1-scale-test",
+		HostnamePrefix:          "auto-rke1-scale-" + ClusterID,
 		NodeTemplateID:          NodeTemplateID,
 		Quantity:                1,
 		Worker:                  true,
 	}
 
+	logrus.Infof("Creating new worker node pool...")
 	nodePool, err := client.Management.NodePool.Create(&nodePoolConfig)
 	if err != nil {
 		return err
 	}
 
-	IsNodeReady(client, nodePool, ClusterID)
+	if nodestat.IsNodeReady(client, ClusterID) != nil {
+		return err
+	} else {
+		logrus.Infof("New node pool is ready!")
+	}
 
 	nodePoolConfig.Quantity = 2
 
+	logrus.Infof("Scaling node pool to 2 worker nodes...")
 	updatedPool, err := client.Management.NodePool.Update(nodePool, &nodePoolConfig)
 	if err != nil {
 		return err
 	}
 
-	IsNodeReady(client, updatedPool, ClusterID)
+	if nodestat.IsNodeReady(client, ClusterID) != nil {
+		return err
+	} else {
+		logrus.Infof("Node pool is scaled to 2 worker nodes!")
+	}
 
 	nodePoolConfig.Quantity = 1
 
+	logrus.Infof("Scaling node pool back to 1 worker node...")
 	_, err = client.Management.NodePool.Update(updatedPool, &nodePoolConfig)
 	if err != nil {
 		return err
+	} else {
+		logrus.Infof("Node pool is scaled back to 1 worker node!")
```

Comment:
Quick nit here Markus, does this else statement necessary? If the error check is not nil, then the function will _return_ the error, and will not continue. Applicable also for this function's error if-else statements.

Reply:
I had the else statements for the sake of logging to help make each step clear while running. Since before, it wasn't necessarily clear what the tests were doing, particularly if you're newer to the test.

Reply:
What Ugur is saying is that the else statement isn't necessary. The logging will still occur even without the else statement. If there is no error it will log and then continue. Make sense?

Reply:
I think so, so you're saying just remove the else and keep the logging? I'll run a quick test of that and make an update

Reply:
Yes exactly, keep the logging but remove the else. Thank you!!!

Reply:
Thanks guys for that catch! I went ahead and updated 👍🏿